### DISCUSSION TITLE
[Question]:  Is there a way to disable/hide the filmstrip?

### DISCUSSION THREAD

**Discussion by JoFranke** [blakeblackshear/frigate#12914] [Ask A Question] [answered]
### What is your question:

Is there a way to disable/hide the filmstrip? Its kind of disturbing when working in a multi monitor environment so that would be really helpful.

Thanks so much for your great work! 

Greetings

**Comment by banthungprong** [comment, marked as answer]
No. See hawkeye's comment in 
https://github.com/blakeblackshear/frigate/discussions/11857
